ui/src/app/subnets/views/SubnetDetails/SubnetSummary/SubnetSummary.tsx

```diff
@@ -1,7 +1,233 @@
+import { useEffect } from "react";
+
+import { Row, Col, Tooltip, Icon } from "@canonical/react-components";
+import { useSelector, useDispatch } from "react-redux";
+
 import TitledSection from "app/base/components/TitledSection";
+import { actions as fabricActions } from "app/store/fabric";
+import fabricSelectors from "app/store/fabric/selectors";
+import { getFabricById } from "app/store/fabric/utils";
+import { actions as spaceActions } from "app/store/space";
+import spaceSelectors from "app/store/space/selectors";
+import { getSpaceById } from "app/store/space/utils";
+import type { Subnet } from "app/store/subnet/types";
+import { actions as vlanActions } from "app/store/vlan";
+import vlanSelectors from "app/store/vlan/selectors";
+import { getVlanById } from "app/store/vlan/utils";
+
+type Props = {
+  subnet: Subnet;
+};
+
+const SubnetSummary = ({ subnet }: Props): JSX.Element => {
+  const dispatch = useDispatch();
+  const spaces = useSelector(spaceSelectors.all);
+  const vlans = useSelector(vlanSelectors.all);
+  const fabrics = useSelector(fabricSelectors.all);
+  const space = getSpaceById(spaces, subnet.space) || null;
+  const vlan = getVlanById(vlans, subnet.vlan) || null;
+  const fabric = vlan ? getFabricById(fabrics, vlan.fabric) : null;
+
+  useEffect(() => {
+    dispatch(spaceActions.fetch());
+    dispatch(vlanActions.fetch());
+    dispatch(fabricActions.fetch());
+  }, [dispatch]);
 
-const SubnetSummary = (): JSX.Element => (
-  <TitledSection title="Subnet summary" />
-);
+  return (
+    <TitledSection title="Subnet summary">
+      <Row>
+        <Col size={6}>
+          <Row>
+            <Col size={2}>
+              <p className="u-text--muted" id="subnet-name">
```

Comment:
We have a `Definition` component that places the label above the content so that it's in the same place as our form fields e.g. https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/subnets/views/VLANDetails/VLANSummary/VLANSummary.tsx#L56